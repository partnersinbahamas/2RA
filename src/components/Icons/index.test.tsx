import * as Icons from './index';

test('Icons', () => {
  expect(Icons).toMatchSnapshot();
});
