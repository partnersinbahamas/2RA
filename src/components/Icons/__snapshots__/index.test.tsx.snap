// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons 1`] = `
Object {
  "AddCircleIcon": [Function],
  "AddIcon": [Function],
  "ArrowBackIcon": [Function],
  "ArrowClockIcon": [Function],
  "ArrowDoubleIcon": [Function],
  "ArrowDownIcon": [Function],
  "ArrowLongDownIcon": [Function],
  "ArrowUpIcon": [Function],
  "AttachmentIcon": [Function],
  "CalculatorIcon": [Function],
  "CalendarIcon": [Function],
  "CancelCircleIcon": [Function],
  "CancelCircleSolidIcon": [Function],
  "CancelIcon": [Function],
  "ChatIcon": [Function],
  "CheckCircleIcon": [Function],
  "CheckCircleSolidIcon": [Function],
  "CheckIcon": [Function],
  "ChevronDownIcon": [Function],
  "ChevronLeftIcon": [Function],
  "ChevronRightIcon": [Function],
  "ChevronUpIcon": [Function],
  "ClockIcon": [Function],
  "CloseIcon": [Function],
  "CopyDocumentIcon": [Function],
  "CopyIcon": [Function],
  "CopyIconSimple": [Function],
  "DBCallIcon": [Function],
  "DataTreeIcon": [Function],
  "DeleteIcon": [Function],
  "DetachIcon": [Function],
  "DocumentIcon": [Function],
  "DownloadCSVIcon": [Function],
  "DownloadIcon": [Function],
  "DownloadXLSIcon": [Function],
  "EditIcon": [Function],
  "EditIconOutlined": [Function],
  "ErrorTriangleIcon": [Function],
  "EuroIcon": [Function],
  "ExclamationCommentIcon": [Function],
  "ExclamationMarkIcon": [Function],
  "ExternalLinkIcon": [Function],
  "FilterIcon": [Function],
  "FullscreenExitIcon": [Function],
  "HelpCircleIcon": [Function],
  "HelpIcon": Object {
    "$$typeof": Symbol(react.memo),
    "compare": null,
    "type": [Function],
  },
  "ImageIcon": Object {
    "$$typeof": Symbol(react.memo),
    "compare": null,
    "type": [Function],
  },
  "InfoIcon": [Function],
  "LightbulbIcon": [Function],
  "LocalTrainIcon": [Function],
  "LockIcon": [Function],
  "MagnifierIcon": [Function],
  "MailIcon": [Function],
  "MapIcon": [Function],
  "MinusCircleIcon": [Function],
  "MinusIcon": [Function],
  "NoTelemetryDataIcon": [Function],
  "NoTrainIcon": [Function],
  "OpenPageIcon": [Function],
  "PercentIcon": [Function],
  "PrintIcon": [Function],
  "RefreshIcon": [Function],
  "ReloadIcon": [Function],
  "ReplaceIcon": [Function],
  "ReportIcon": [Function],
  "SaveIcon": [Function],
  "SendIcon": [Function],
  "ServiceIcon": [Function],
  "SettingsIcon": [Function],
  "SettingsPinIcon": [Function],
  "ShareIcon": [Function],
  "SheetClockIcon": [Function],
  "SheetProgressIcon": [Function],
  "SlashIcon": [Function],
  "SmoothUploadIcon": [Function],
  "StarIcon": [Function],
  "SwapHorizontalIcon": [Function],
  "UnlinkIcon": [Function],
  "UnlockIcon": [Function],
  "UploadIcon": [Function],
  "VisibilityOffIcon": [Function],
  "VisibilityOnIcon": [Function],
  "WarnIcon": [Function],
}
`;
